pkg/core/interop_neo.go

```diff
@@ -82,11 +83,22 @@ func (s *storageIterator) Value() stackitem.Item {
 	if s.opts&FindKeysOnly != 0 {
 		return s.m[s.index].Key
 	}
+	value := s.m[s.index].Value
+	if s.opts&FindDeserialize != 0 {
+		bs, err := s.m[s.index].Value.TryBytes()
+		if err != nil {
+			panic(err) // impossible due to how map is created
+		}
```

Comment:
Then it might be just `bs := s.m[s.index].Value.Value().([]byte)`

Reply:
Fixed.